libs/abis/project.json

```diff
@@ -0,0 +1,33 @@
+{
+  "root": "libs/abis",
+  "sourceRoot": "libs/abis/src",
+  "projectType": "library",
+  "targets": {
+    "build": {
+      "executor": "@nrwl/js:tsc",
+      "outputs": ["{options.outputPath}"],
+      "options": {
+        "outputPath": "dist/libs/abis",
+        "main": "libs/abis/src/index.ts",
+        "tsConfig": "libs/abis/tsconfig.lib.json",
+        "assets": ["libs/abis/*.md"]
+      }
+    },
+    "lint": {
+      "executor": "@nrwl/linter:eslint",
+      "outputs": ["{options.outputFile}"],
+      "options": {
+        "lintFilePatterns": ["libs/abis/**/*.ts"]
+      }
+    },
+    "test": {
+      "executor": "@nrwl/jest:jest",
+      "outputs": ["coverage/libs/abis"],
+      "options": {
+        "jestConfig": "libs/abis/jest.config.js",
+        "passWithNoTests": true
+      }
+    }
+  },
```

Comment:
Do we want to release this as an npm pakage?

Reply:
I'm not 100% sure. 

Our other packages/apps will use it for sure.

It might be nice? What do you think?

Reply:
We can release, I don't see any reason not too